### PR TITLE
Doc Updates

### PR DESCRIPTION
**Pull request by quasiben**
Resolves https://github.com/dask/dask-kubernetes/issues/144

This is a first pass at getting a working example and updating docs on using dask-kubernetes with GPUs (this has been coming up a more for me and having some docs I can point to would be very valuable)

Also adds docs on remote deploys

### CODE REVIEW

File: doc/source/kubecluster.rst [hunk 1]
```diff
@@ -192,4 +225,45 @@ The pods are created with two default `tolerations <https://kubernetes.io/docs/c
...
+
+When deploying remotely, the following k8s resources are created:
+
+- A pod with a scheduler running
+- (optional) A pod with a LoadBalancer and complimentary service (svc) to
+  expose scheduler and dashobard ports
+
+By default, the ``scheduler-service-type`` is set to ``ClusterIp``. To
+optionally use the LoadBalancer, change ``scheduler-service-type`` to
+``LoadBalancer``
```
Comment (holdenk) [inline doc/source/kubecluster.rst:268]: Where should folks set this? Looking at the code it seems like it's pulled from `kubernetes.yaml`, which does not seem to be configurable.

Comment (holdenk) [reply doc/source/kubecluster.rst:268]: Is it with `dask.config`?

Comment (holdenk) [reply doc/source/kubecluster.rst:268]: Answered my own question, was with `dask.config.set({"kubernetes.scheduler-service-type": "LoadBalancer"})`, maybe we can add that?

Comment (quasiben) [reply doc/source/kubecluster.rst]: Thanks the for the suggestion @holdenk.  Added your thoughts in the last commit

**Review by jacobtomlinson — APPROVED** [review]
Thanks @quasiben

**Pull request merged**